airbyte-integrations/connectors/source-jira/source_jira/streams.py

```diff
@@ -857,6 +863,68 @@ def read_records(self, stream_slice: Optional[Mapping[str, Any]] = None, **kwarg
             yield from super().read_records(stream_slice={"key": project["key"]}, **kwargs)
 
 
+class PullRequests(IncrementalJiraStream):
+    """
+    This stream uses an undocumented internal API endpoint used by the Jira
+    webapp. Jira does not publish any specifications about this endpoint, so the
+    only way to get details about it is to use a web browser, view a Jira issue
+    that has a linked pull request, and inspect the network requests using the
+    browser's developer console.
+    """
+
+    cursor_field = "updated"
+    parse_response_root = "detail"
+    raise_on_http_errors = False
+
+    pr_regex = r"(?P<prDetails>PullRequestOverallDetails{openCount=(?P<open>[0-9]+), mergedCount=(?P<merged>[0-9]+), declinedCount=(?P<declined>[0-9]+)})|(?P<pr>pullrequest={dataType=pullrequest, state=(?P<state>[a-zA-Z]+), stateCount=(?P<count>[0-9]+)})"
+
+    def __init__(self, issues_stream: Issues, issue_fields_stream: IssueFields, **kwargs):
+        super().__init__(**kwargs)
+        self.issues_stream = issues_stream
+        self.issue_fields_stream = issue_fields_stream
+
+    @property
+    def url_base(self) -> str:
+        return f"https://{self._domain}/rest/dev-status/1.0/"
+
+    def path(self, **kwargs) -> str:
+        return f"issue/detail"
```

Comment:
```suggestion
        return "issue/detail"
```
remove f-string is breaking the format